### PR TITLE
Feature/restaurant page backend

### PR DESCRIPTION
**Pull request by carlosshi1**
Updated the restaurant view and connected it with the search page

### CODE REVIEW

File: pages/restaurant/[id].js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      <div className={styles.contentContainer}>
+        <ImageHeader />
+
+        <div className={styles.contentBody}>
+          <div>
+            <Title>{resto.name}</Title>
+            <Rate allowHalf value={resto.averageRating} disabled />
+            &nbsp;&nbsp;&nbsp;
+            {resto.averageRating}
+          </div>
```
Comment (NeilMatthewLua) [inline pages/restaurant/[id].js]: Let's comment this out first since we don't have reviews yet

File: components/restoprofile/BasicInfo.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+              </div>
+            }
+          >
+            {resto.establishmentType}
+          </Descriptions.Item>
+          <Descriptions.Item
+            label={
+              <div>
+                <MoneyCollectOutlined />
+                &nbsp;Avg. Cost
```
Comment (NeilMatthewLua) [inline components/restoprofile/BasicInfo.js]: Avg. Cost for Two*

File: pages/restaurant/[id].js (hunk 1 above)
Comment (carlosshi1) [reply pages/restaurant/[id].js]: We do have ratings

File: components/restoprofile/BasicInfo.js (hunk 2 above)
Comment (carlosshi1) [reply components/restoprofile/BasicInfo.js]: ok will fix this one

**Review by NeilMatthewLua — APPROVED** [review]

**Pull request merged**